ocaml/xapi/import.ml

```diff
@@ -275,11 +275,15 @@ let assert_can_live_import __context rpc session_id vm_record =
            )
         )
   in
-  if
-    vm_record.API.vM_power_state = `Running
-    || vm_record.API.vM_power_state = `Paused
-  then
-    assert_memory_available ()
+  match vm_record.API.vM_power_state with
```

Comment:
I think we don't need this change anymore now, as the changes below apply to but live and non-live?

Reply:
We need this match because based on power state we need to allocate memory or not. It was previously expressed with `if` and is now expressed with `match`.

Reply:
Yes, we need the old code. Changing it seems unnecessary, and the comment about DMC below misleading.